### PR TITLE
Massive MEP move

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: doc/devel/MEP/MEP09.rst [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+The current MEP also includes grab handles for resizing artists, and
+appropriate boxes drawn when artists are moved or resized.
+
+Implementation
+==============
+* Add appropriate methods to the "tree" of artists so that the
+  interactivity manager has a consistent interface for the
+  interactivity manager to deal with.  The proposed methods to add to
+  the artists, if they are to support interactivity, are:
+    * get_pixel_position_ll(self): get the pixel position of the lower
```
Comment (efiring) [inline doc/devel/MEP/MEP09.rst]: It looks like Travis is complaining about the indentation here.

**Comment by tacaswell** [comment]
This is incredibly frustrating to test locally as building the docs seems to take way longer on my machine than travis (which probably means I need a new desktop...).

**Comment by fariza** [comment]
@tacaswell do you have the URL to see the MEP documents that are already merged?

**Comment by tacaswell** [comment]
@fariza http://matplotlib.org/devdocs/devel/MEP/index.html

Those get updated everytime we have a passing build of the master branch on travis.

**Comment by fariza** [comment]
We should change the wiki page, pointing to that new page, and a little explanation on how this is going to work from now on.

**Comment by tacaswell** [comment]
Yes, that is (too far down) on my internal to-do.

**Pull request merged**